src/fidesops/task/graph_task.py

```diff
@@ -526,11 +563,14 @@ def termination_fn(*dependent_values: List[Row]) -> Dict[str, List[Row]]:
         env: Dict[CollectionAddress, Any] = {}
         end_nodes = traversal.traverse(env, collect_tasks_fn)
 
-        dsk = {k: (t.access_request, *t.input_keys) for k, t in env.items()}
-        dsk[ROOT_COLLECTION_ADDRESS] = (start_function(traversal.seed_data),)
+        dsk: Dict[CollectionAddress, Tuple[Any, ...]] = {
+            k: (t.access_request, *t.input_keys) for k, t in env.items()
+        }
+        dsk[ROOT_COLLECTION_ADDRESS] = (start_function([traversal.seed_data]),)
         dsk[TERMINATOR_ADDRESS] = (termination_fn, *end_nodes)
-        v = dask.delayed(get(dsk, TERMINATOR_ADDRESS))
+        update_mapping_from_cache(dsk, resources, start_function)
 
+        v = dask.delayed(get(dsk, TERMINATOR_ADDRESS, num_workers=1))
```

Comment:
This is another primary change.  Using one worker on the threaded scheduler will force our tasks to run sequentially for a given privacy request.  

For some background, Pausing a privacy request throws an exception to stop execution. Dask Delayed assumes that every other task should stop.   Running sequentially here helps make sure a closed session doesn't affect other tasks that are in flight, as well as makes sure all execution logs get written in full.  With this design, only one collection can be paused at one time for a privacy request.  There's pros and cons with this, but I think it's the cleanest way to stop PrivacyRequest execution with Dask Delayed.

This also feels more similar to Sean's proposed design too which sequentially ordered these tasks in Celery.